app/lib/three_scale/analytics/user_tracking.rb

```diff
@@ -211,7 +211,7 @@ def cache_key
       end
 
       def can_send?
-        user_id && @account.try!(:provider?) && user_type != 'impersonation_admin'
+        !ThreeScale.config.onpremises && user_id && @account.try!(:provider?) && user_type != 'impersonation_admin'
```

Comment:
Can we know if Segment is properly configured instead of using this (do not use anymore `ThreeScale.config.onpremises`) ?

Reply:
@hallelujah maybe pinging segment only once to check if it is enabled?

Reply:
no, it is easier 😄 but i didn't have time to explain yet 😂

Reply:
I will explain in the sync @duduribeiro